src/View/Form/ContextInterface.php

```diff
@@ -48,9 +48,14 @@ public function isCreate();
      *
      * @param string $field A dot separated path to the field a value
      *   is needed for.
+     * @param array $options Options:
+     *   - `default`: Default value to return if no value found in request
+     *     data or context record.
+     *   - `schemaDefault`: Boolen indicating whether default value from
+     *      context's schema should be used if it's not explicitly provided.
      * @return mixed
      */
-    public function val($field);
+    public function val($field, $options = []);
```

Comment:
PHP doesn't seem to complain if the `$options` param is removed from interface method. So can we have it in implementing classes only?  The classes would have to be updated to account for the options but at least there won't be any signature mismatch errors until they are updated.